quickstarts/maven/spring-boot/pom.xml

```diff
@@ -36,7 +36,7 @@
   </description>
 
   <properties>
-    <jkube.version>${project.version}</jkube.version>
+    <jkube.version>1.17-SNAPSHOT</jkube.version>
```

Comment:
Could you please revert this to what it was before?

Reply:
Yes of course